src/features/auth/pages/signup/SetEmail/SetEmail.native.test.tsx

```diff
@@ -301,6 +309,37 @@ describe('<SetEmail />', () => {
         accountCreationToken: 'accountCreationToken',
       })
     })
+
+    it('should display snackbar when SSO account is invalid', async () => {
+      mockServer.getApiV1<OauthStateResponse>('/oauth/state', {
+        oauthStateToken: 'oauth_state_token',
+      })
+      mockServer.postApiV1<SignInResponseFailure['content']>('/oauth/google/authorize', {
+        responseOptions: {
+          statusCode: 401,
```

Comment:
Si on veut être vraiment précis sur le code renvoyé par le backend pour cette erreur
```suggestion
          statusCode: 400,
```